### ISSUE TITLE
Spellchecker error with "en_NZ" locale

### ISSUE THREAD

**Issue by sebastien** [lmorg/murex#349]
Hi there,

I'm in NZ and my locale is "en_NZ", which cause a systematic error on every murex prompt:

```
~ »                                                                             
Spellchecker produced an error: STDERR: Error: No word lists can be found for the language "en_NZ".     
```

Installing `aspell-en` did fix the problem, but I would recommend that the error message is only displayed once and that it indicates that `aspell` should be installed.

**Comment by lmorg** [comment]
Hi @sebastien 

Thanks for raising this issue.

The error being generated is from `aspell` itself rather than _murex_ and the spellchecker function is a _murex_ function (designed this way so users can chose alternative spellchecker, like `hunspell`, if preferred):
```
» config get shell spellcheck-block
{ -> aspell list }
```
(**note to self:** I should change that to `spellcheck-func` to bring it in line with the other function keys like `hint-text-func`)

But I can definitely do a little better than pass that STDERR message verbatim...

The commit above (5bbd0ee) adds a little more context to the error message to say where the error has come from and includes a URL to the _murex_ doc (https://murex.rocks/docs/user-guide/spellcheck.html) which hosts spellcheck instructions. I should add more detail on how to fix common issues there too (this will be added shortly).

Regarding your suggestion of checking if `aspell` is installed: _murex_ does already disable spellchecking if `aspell` isn't installed but in this case `aspell` was available in `$PATH` but it needed further configuration. Unfortunately I cannot detect if a dependency or configuration for `aspell` is missing without running `aspell` itself and if I run `aspell` then I need to pass that error to the user -- because that's who would need to configure `aspell` -- and that's what was already happening here. So hopefully the additionally information I've added to the error message adds a little more context and detail to the user so they're better able to diagnose the problem and so the error is less confusing.

Displaying the error only once is a good suggestion though. I've also included that fix in the commit above (5bbd0ee) and like with the more descriptive error message, that will be incorporated in the next release of _murex_ (v2.2).

Thanks again for your support in raising issues.
